internal/utils/tdl.go

```diff
@@ -168,6 +168,11 @@ func ConvertTwitchLiveChatToTDLChat(path string, channelName string, videoID str
 			},
 		}
 
+		if (liveComment.MessageType == "highlighted_message") {
+			var highlightString = "highlighted-message"
+			tdlComment.Message.UserNoticeParams.MsgID == &highlightString
```

Comment:
```suggestion
			tdlComment.Message.UserNoticeParams.MsgID = &highlightString
```
Just need one `=` here as it's being assigned to the variable.

Reply:
Whoops, didn't mean to type a double `=` there